serverless-functions/mozilliansorg_groups_publisher/handler.py

```diff
@@ -0,0 +1,30 @@
+import cis_logger
+import logging
+import socket
+import sys
+
+import cis_publisher
+from aws_xray_sdk.core import xray_recorder
+from aws_xray_sdk.core import patch_all
+
+
+patch_all()
+mozilliansorg_groups = cis_publisher.mozilliansorg_groups.MozilliansorgGroupsPublisher()
+
+
+def setup_logging():
+    logger = logging.getLogger()
+    for h in logger.handlers:
+        logger.removeHandler(h)
+    h = logging.StreamHandler(sys.stdout)
+    h.setFormatter(cis_logger.JsonFormatter(extra={"hostname": socket.gethostname()}))
```

Comment:
I forgot if we do this elsewhere, but the extra should probably be defaulted in `cis_logger`

Reply:
extra defaults to `{}` if I see this right.